deployment/migrations/scripts/0001-update-keys-for-p2pd.py

```diff
@@ -0,0 +1,63 @@
+"""
+This migration moves the private key file to the key directory and generates two new keys:
+a public key, and a serialized version of the private key for use by the P2P service.
+"""
+
+
+import logging
+import os
+
+from Crypto.PublicKey import RSA
+from p2pclient.libp2p_stubs.crypto.rsa import KeyPair, RSAPrivateKey
+
+from aleph.exceptions import InvalidKeyDirException
+from aleph.services.keys import save_keys
+
+LOGGER = logging.getLogger("migrations")
+
+
+SERIALIZED_KEY_FILE = "serialized-node-secret.key"
+
+
+def populate_key_dir(key_file: str, output_dir: str) -> None:
+    with open(key_file) as f:
+        private_key_str = f.read()
+
+    private_key = RSAPrivateKey(RSA.import_key(private_key_str))
+    key_pair = KeyPair(private_key=private_key, public_key=private_key.get_public_key())
+    save_keys(key_pair, output_dir)
+
+
+def upgrade(**kwargs):
+    key_dir = kwargs["key_dir"]
```

Comment:
Using `pathlib.Path` can make joining paths easier: https://docs.python.org/3/library/pathlib.html , especially for joining using `/`, checking if a file exists, ...

```suggestion
    from pathlib import Path
    key_dir = Path(kwargs["key_dir"])
```

Reply:
I prefer the old school way of manipulating paths, but sure if you prefer. Not a big deal.

Reply:
I don't always have the reflex neither, but I think it would be useful here.